sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/MutatorQueryFunctionTest.kt

```diff
@@ -11,31 +12,31 @@ class MutatorQueryFunctionTest {
   @get:Rule val tempFolder = TemporaryFolder()
 
   @Test fun `mutator method generates proper method signature`() {
-    val file = FixtureCompiler.parseSql("""
-      |CREATE TABLE data (
-      |  id INTEGER NOT NULL PRIMARY KEY,
-      |  value TEXT AS kotlin.collections.List
-      |);
-      |
-      |insertData:
-      |INSERT INTO data
-      |VALUES (?, ?);
-      """.trimMargin(), tempFolder)
-
-    val insert = file.namedMutators.first()
-    val generator = MutatorQueryGenerator(insert)
-
-    assertThat(generator.function().toString()).isEqualTo("""
-      |override fun insertData(id: kotlin.Long?, value: kotlin.collections.List?) {
-      |  driver.execute(${insert.id}, ""${'"'}
-      |  |INSERT INTO data
-      |  |VALUES (?, ?)
-      |  ""${'"'}.trimMargin(), 2) {
-      |    bindLong(1, id)
-      |    bindString(2, if (value == null) null else database.dataAdapter.valueAdapter.encode(value))
-      |  }
-      |}
-      |""".trimMargin())
+    for (dialectPreset in listOf(DialectPreset.SQLITE_3_18, DialectPreset.MYSQL)) {
```

Comment:
I will add a rule for this eventually, but currently there's some nullability issues that cause the output to be different between MySQL and SQLite, and I didn't think it was worth blocking this fix.